mailer.js

```diff
@@ -11,6 +12,24 @@ var DEFAULT_LOCALE = 'en'
 var DEFAULT_TIMEZONE = 'Etc/UTC'
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var templateNameToCampaignMap = {
+    'passwordResetRequiredEmail': 'password-reset-required',
+    'passwordChangedEmail': 'password-changed-success',
+    'passwordResetEmail': 'password-reset',
+    'postVerifyEmail': 'account-verified',
+    'newDeviceLoginEmail': 'new-device-signin',
+    'recoveryEmail': 'forgot-password',
```

Comment:
There is no difference, looks like this was a case of differences in mailer function and template names. Renaming `reset.html` to `recovery.html` clears up discrepancy. Will refactor template name.